src/game.cpp

```diff
@@ -9144,7 +9144,17 @@ bool game::walk_move( const tripoint &dest_loc )
     }
     u.set_underwater( false );
 
-    if( !shifting_furniture && !pushing && !prompt_dangerous_tile( dest_loc ) ) {
+    if( !shifting_furniture && !pushing && is_dangerous_tile( dest_loc ) &&
+        !u.movement_mode_is( CMM_RUN ) ) {
+        std::vector<std::string> harmful_stuff = get_dangerous_tile( dest_loc );
+        std::string key_message = press_x( ACTION_TOGGLE_RUN );
+        if( key_message == ( "Try" ) ) {
```

Comment:
`press_x` returns a translated string. You are comparing this with a fixed string. That won't work when the user has a non-English language set.